akvo/rsr/dir/app/modules/project-page/components/Disaggregations.jsx

```diff
@@ -2,58 +2,20 @@ import React, { useEffect, useState } from 'react'
 import { Col, Divider, Row, Skeleton, Typography } from 'antd'
 import groupBy from 'lodash/groupBy'
 import sumBy from 'lodash/sumBy'
-import api from '../../../utils/api'
 import { setNumberFormat } from '../../../utils/misc'
 
 const { Text } = Typography
 
-const Disaggregations = ({
-  id,
-  updates,
```

Comment:
You can do like this `updates = []` so that if updates is null or undefined it will not break anything

Reply:
ah okay, Thanks @navins94 . I'm working to fix it